lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -16,7 +16,7 @@ module AccessTokenMixin
       validates :token, presence: true, uniqueness: true
       validates :refresh_token, uniqueness: true, if: :use_refresh_token?
 
-      attr_writer :use_refresh_token
+      attr_writer :use_refresh_token, :use_unique_refresh_token_index?
```

Comment:
Do we need to use `use_unique_refresh_token_index?` here, or in https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/orm/mongoid4/access_token.rb#L21?